mongoengine/connection.py

```diff
@@ -118,8 +118,6 @@ def get_connection(alias=DEFAULT_CONNECTION_NAME, reconnect=False):
             conn_settings.pop('name', None)
             conn_settings.pop('slaves', None)
             conn_settings.pop('is_slave', None)
-            conn_settings.pop('username', None)
-            conn_settings.pop('password', None)
```

Comment:
Just to make sure I understand correctly: Since we're no longer popping the `username` & the `password` off of `conn_settings`, they are being passed into the `MongoClient` below (via `connection_class(**conn_settings)`).

Right?

Reply:
Yep! We pass the credentials through to `MongoClient` as `username` and `password` parameters now instead of using the separate `db.authenticate` call.